export_kakuma_advert_contacts.py

```diff
@@ -0,0 +1,173 @@
+    # i.e Oromo/Sudanese-Juba arabic/Somali/Swahili speakers.
+    oromo_uuids = set()
+    sudanese_juba_arabic_uuids = set()
+    turkana_uuids = set()
+    somali_uuids = set()
+    english_uuids = set()
+    swahili_uuids = set()
+    all_uids = set()
+
+    log.info(f'Searching for the participants uuids vis-a-vis` their manually labelled '
+             f'demographic language response')
+    for msg in messages:
+        if msg['uid'] in all_uids or msg["consent_withdrawn"] == Codes.TRUE:
+            continue
+
+        if CodeSchemes.KAKUMA_HOUSEHOLD_LANGUAGE.get_code_with_code_id(
+                msg["household_language_coded"]["CodeID"]).string_value == "oromo":
+            oromo_uuids.add(msg['uid'])
+            all_uids.add(msg['uid'])
+        elif CodeSchemes.KAKUMA_HOUSEHOLD_LANGUAGE.get_code_with_code_id(
+                    msg["household_language_coded"]["CodeID"]).string_value == "sudanese":
+                sudanese_juba_arabic_uuids.add(msg['uid'])
+                all_uids.add(msg['uid'])
+        elif CodeSchemes.KAKUMA_HOUSEHOLD_LANGUAGE.get_code_with_code_id(
+                    msg["household_language_coded"]["CodeID"]).string_value == "turkana":
+                turkana_uuids.add(msg['uid'])
+                all_uids.add(msg['uid'])
+        elif CodeSchemes.KAKUMA_HOUSEHOLD_LANGUAGE.get_code_with_code_id(
+                    msg["household_language_coded"]["CodeID"]).string_value == "somali":
+                somali_uuids.add(msg['uid'])
+                all_uids.add(msg['uid'])
+        elif CodeSchemes.KAKUMA_HOUSEHOLD_LANGUAGE.get_code_with_code_id(
+                    msg["household_language_coded"]["CodeID"]).string_value == "english":
+                english_uuids.add(msg['uid'])
+                all_uids.add(msg['uid'])
+        else:
+            swahili_uuids.add(msg['uid'])
+            all_uids.add(msg['uid'])
+
+    # Convert the uuids to phone numbers
+    log.info("Converting the uuids to phone numbers...")
+    uuids_to_phone_numbers = phone_number_uuid_table.uuid_to_data_batch(list(all_uids))
+
+    oromo_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in oromo_uuids]
+    sudanese_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in sudanese_juba_arabic_uuids]
+    turkana_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in turkana_uuids]
+    somali_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in somali_uuids]
+    english_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in english_uuids]
+    swahili_phone_numbers = [f"+{uuids_to_phone_numbers[uuid]}" for uuid in swahili_uuids]
+
+    # Export the phone numbers to the advert CSV
+    # TODO upload this to keep_ii_kakuma textit instance through API?
+    advert_contacts = OrderedDict()
+    for contact_list in [oromo_phone_numbers, sudanese_phone_numbers, turkana_phone_numbers, somali_phone_numbers,
```

Comment:
This construct is unnecessarily convoluted. Instead of:
```
for contact_list in ...:
    if contact_list == ...:
        for phone_number in contact_list:
           ...
```

consider:
```
for phone_number in oromo_phone_numbers:
   ...

for phone_number in sudanese_phone_numbers:
   ...
```
etc.

Reply:
updated thanks!